core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1245,9 +1250,8 @@ class DAGScheduler(
                     markMapStageJobAsFinished(job, stats)
                   }
                 }
+                submitWaitingChildStages(shuffleStage)
```

Comment:
Do we need to call this `submitWaitingChildStages(shuffleStage)`  when the given `shuffleStage` is not available?